src/filtering/filterDialogue.component.tsx

```diff
@@ -435,57 +496,58 @@ const FilterDialogue = (props: FilterDialogueProps) => {
       </DialogContent>
       <DialogActions>
         <Button onClick={handleClose}>Close</Button>
-        {displayingWarningMessage ? (
-          <Tooltip
-            componentsProps={{
-              tooltip: {
-                sx: {
-                  backgroundColor: 'yellow',
-                  color: 'black',
-                  border: '1px solid black',
+        {tabValue !== 'Favourite filters' &&
```

Comment:
Nothing dangerous about this in the current PR, but in future PRs where you can apply a favourite filter from the favourite filters panel then this could be problematic... Just making a comment so it sticks in my mind more to check for this in the future PR :)